moped-database/migrations/1723496266686_add_bike_share_station_comp/up.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO moped_components (
+    component_name, line_representation, feature_layer_id
+) VALUES
+('Bike Share Station', FALSE, 5);
```

Comment:
@mddilley i think you'll need to add some more inserts here to associate this component with the work types and subcomponent (if any) that should be available int he UI. e.g., it has no work types available currently:

<img width="594" alt="Screenshot 2024-08-14 at 12 58 12 PM" src="https://github.com/user-attachments/assets/d444af64-bf58-4c4b-8ce5-e611264fd79f">

this migration probably has some boilerplate you can copy:
https://github.com/cityofaustin/atd-moped/blob/main/moped-database/migrations/1698950636188_create_shared_use_trail_dual_path_component/up.sql

Reply:
yep, i'll update this so that the **required** field can be filled out. 🤦‍♂️  Thanks for catching this because I only thought about the subcomponents, and I forgot that the work types worked this way. 🙏 🙌